.travis.yml

```diff
@@ -41,24 +41,24 @@ jobs:
       stage: pre-test
       script: ./scripts/ci/ci_run_all_static_checks.sh
       env: >-
-        PYTHON_VERSION=3.6
+        PYTHON_MAJOR_MINOR_VERSION=3.6
```

Comment:
Why do we need to change it? I don't understand what means "major_minor" :<

Reply:
I already explained it before - because  PYTHON_VERSION is set to 3.6.9 in the base Python images and it causes a lot of confusion as depending on the script sometimes you can get PYTHON_VERSION = 3.6.9. 

MAJOR_MINOR_VERSION is pretty straightforward IMHO. Do you have any other proposal. The requirements for the name are:

1) can't be PYTHON_VERSION (already set by the base image)
2) Has to correctly reflect 3.9 being MAJOR.MINOR version of python
3) Cannot have "." (environment variable can't have dots).

What is your proposal for the name of this variable please ?

Reply:
Thanks!